docs/src/config.md

```diff
@@ -282,8 +286,11 @@ with `only` as the default. To extract data for a specific location (grid cell),
 of the vector, the coordinates `coordinate.x` and `coordinate.y`, or the x and y indices of
 the 2D array (`index.x` and `index.y`) can be provided. Finally a `map` can be provided to
 extract data for certain locations (e.g. `gauges`) or areas (e.g. `subcatchment`). In this
-case a single entry can lead to multiple columns in the CSV file, which will be of the
-form `header_id`, e.g. `Q_20`, for a gauge with integer ID 20.
+case a single entry can lead to multiple columns in the CSV file, which will be of the form
+`header_id`, e.g. `Q_20`, for a gauge with integer ID 20. Model parameters with the extra
+dimension `layer` for layered model parameters of the vertical `sbm` concept require the
+specification of the layer (see also example below). If multiple layers are desired, this
```

Comment:
should we specify in the documentation here: "the specification of the (Julia) index of the layer"?
in staticmaps, variable c has layer coordinates of [0,1,2,3], and in order to get c_0, the config should be:

```
[[csv.column]]
coordinate.x = 6.255
coordinate.y = 50.012
header = "vwc_layer0_bycoord"
parameter = "vertical.vwc"
layer = 1
```

but I thought we wanted to link this directly to the name of the layer (in my understanding this would be 0, 1, 2, 3 in this case instead of 1, 2, 3, 4)?

Reply:
Yes, this is indeed how the dimension layer is defined internally in Wflow as part of `sbm`. This is also how we write the `layer` dimension to the gridded NetCDF output ([1,2,3,4]). I agree, would be good to add in the text that this is internal defined.

Reply:
Changed this to `internal layer index` in commit https://github.com/Deltares/Wflow.jl/pull/193/commits/089858161c6397516f5d69cf31bf90e8ade8253f.